locale/en_GB/LC_MESSAGES/djangojs.po

```diff
@@ -1217,30 +1278,40 @@ msgstr ""
 "content will only be loaded after you give your consent. Only then will "
 "YouTube set Cookies that collect information on user behaviour."
 
-#: dsgvo-video-embed/js/dsgvo-video-embed.js:15
+#: dsgvo-video-embed/js/dsgvo-video-embed.js:13
 msgid "More information can be found in the privacy policy of Google under: "
 msgstr "More information can be found in the privacy policy of Google under: "
 
-#: dsgvo-video-embed/js/dsgvo-video-embed.js:17
-#: dsgvo-video-embed/js/dsgvo-video-embed.js:22
+#: dsgvo-video-embed/js/dsgvo-video-embed.js:15
 msgid "Link to Video: "
```

Comment:
here is `Video` and in line 1290 and below is `video`. Does it need to change? Or since site is only in German, it doesn't mind?

Reply:
Should be the same, yeah. Will make an issue at https://github.com/liqd/dsgvo-video-embed